docs/file/lib/tupelo.js.html

```diff
@@ -82,14 +83,28 @@
   });
 };
 
-const CHAINTREE_DATA_PATH = &quot;/tree/data&quot;
+const walletCredMetadata = (walletCreds) =&gt; {
+  var meta = new grpc.Metadata();
+  meta.add(&apos;wallet&apos;, walletCreds.walletName);
+  meta.add(&apos;passphrase&apos;, walletCreds.passphrase);
+
+  return meta;
+};
+
+const connectionCredentials = (secure) =&gt; {
+  if (secure) {
+    return grpc.credentials.createSsl();
+  } else {
+    return grpc.credentials.createInsecure();
+  }
+};
 
 /**
  * Represents a connection to a specific Tupelo wallet managed by a remote
  * Tupelo RPC server
  */
 class TupeloClient {
-  constructor(walletServer, walletCreds) {
+  constructor(walletServer, walletCreds, security = true) {
```

Comment:
Incredibly minor, but I think this arg's purpose would be a little clearer and more consistent with common usage if it were named `secure`.